ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+# 2022-10-16
+
+ - Added a new `flake.flakePartsModules` option so a flake can expose a module
+   to be used in a downstream flake's flake-parts usage.
```

Comment:
`foo.flakePartsModules.default` is going to be quite a mouthful compared to the current `foo.flakeModule`.
On the other hand, `flakeModule` is a holdover from when this project was called `flake-modules`.

Reply:
I didn't know about the holdover. I had always assumed the name `flakeModule` to be appropriate because ... well .. flake-parts is the _only_ known module system (afaict) for flakes. And that, if/when flake-parts get subsumed into nixpkgs proper we would then have an official module system with the added hope of inheriting the `flakeModule` naming *as is*.

Reply:
Switched to `flake.flakeModules` and added `flakeModule` alias.